intel_extension_for_transformers/llm/library/jblas/jblas/kernel_avx2.h

```diff
@@ -50,6 +50,10 @@ static inline __m128i unpack_4bits_sse(void* srcptr) {
 template <JBLAS_SIGN_INT_TYPE S4_T>
 static inline void convert_s4_s8_16_sse(int8_t* dstptr, int8_t* srcptr) {
   auto dst0 = unpack_4bits_sse<S4_T>(srcptr);
+  if constexpr (S4_T == S4_FULLRANGE) {
```

Comment:
do we have some results to show the acc benefits of full range? @xin3he  @zhewang1-intc